deployment/cfn/utils/constants.py

```diff
@@ -1,4 +1,5 @@
 EC2_INSTANCE_TYPES = [
+    't3.micro',
```

Comment:
It's probably worth remaining consistent as single quotes. Sorry for letting this stuff slip through.

Reply:
I think it makes sense to keep `t3.micro` in the `EC2_INSTANCE_TYPES` list, but the bastion is currently still a `t3.micro`. We should convert it back to the T2 instance family because we have 12 RIs within the account (we're going to be over no matter what because the bastion used to be a larger instance type).